.github/styles/Papermoon/WordSwapList.yml

```diff
@@ -0,0 +1,82 @@
+extends: substitution
+message: "Use '%s' instead of '%s'."
+link: "https://developers.google.com/style/word-list"
+level: warning
+ignorecase: false
+action:
+  name: replace
+swap:
+  "(?:API Console|dev|developer) key": API key
```

Comment:
Can you please provide an example of what this does?

Reply:
This swap list is a customization of the Google package one. I haven't really taken things out of it so there are likely things we don't want. 

This one will create a warning for "Use this instead of that". In the API instance, there is scripting so it will catch the different variations (API Console, API dev, etc) and generate a warning to use API key instead. We can safely get rid of anything in this list we don't feel is relevant.

Reply:
So this overlaps with configs already provided by Google? I'd say we should clean this up to only relevant stuff that we want. Checked quickly and these are probably the only relevant ones, so I'd say just keep these unless they overlap with Google configs

"(?:file ?path|path ?name)": path
file name: filename
HTTPs: HTTPS
url: URL
wasm: Wasm

Reply:
@dawnkelly09 this may have gotten overlooked

Reply:
I kept a few others just for little language consistency things. This is the new list I'm proposing. Please let me know if you see anything else that you want removed. Thank you! 
swap:
  "(?:e-mail|Email|E-mail)": email
  "(?:file ?path|path ?name)": path
  "(?:kill|terminate|abort)": stop|exit|cancel|end
  "(?:ok|Okay)": OK|okay
  "(?:WiFi|wifi)": Wi-Fi
  '3\-D': 3D
  "un(?:check|select)": clear
  above: preceding
  admin: administrator
  Ajax: AJAX
  a\.k\.a|aka: or|also known as
  approx\.: approximately
  check box: checkbox
  disabled?: turn off|off|deactivate
  file name: filename
  HTTPs: HTTPS
  in order to: to
  open-source: open source
  stylesheet: style sheet
  tablename: table name
  url: URL
  vs\.: versus
  wasm: Wasm

Reply:
sounds good thank you